Presentation.Web/Models/External/V2/Types/Shared/SimpleLinkDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Presentation.Web.Models.External.V2.Types.Shared
+{
+    public class SimpleLinkDTO
+    {
+        /// <summary>
+        /// Name displayed to the user in kitos
+        /// </summary>
+        public string Name { get; set; }
```

Comment:
Har vi ikke maxlength på denne? Der er lavet en markering i beskrivelsen af en af DTO'erne som benytter denne at maxlength er 150

Reply:
Det er ikke alle simplelink der har den begrændning (når jeg ser vores excel ark igennem.. :( )

Reply:
kan selvfølgelig løses med en custom attribute ligesom med nonEmptyGuid. Det prøver jeg lige for det hjælper jo med input valideringen